api/internal/git/repospec.go

```diff
@@ -197,6 +202,7 @@ func peelQuery(arg string) (string, string, time.Duration, bool) {
 
 func parseHostSpec(n string) (string, string) {
```

Comment:
Technically, this function does have its own unit tests. `parseHostSpec` is essentially passed the raw url and its output is directly stored as `repoSpec.Host`. `TestNewRepoSpecFromUrl` unit tests `parseHostSpec` by checking the value of `rs.Host`. 

If I were to write a separate test for `parseHostSpec`, it would basically do the same thing as `TestNewRepoSpecFromUrl`, without checking the other fields. I added a test case to `TestNewRepoSpecFromUrl` to demonstrate my change. Is it not enough?

Reply:
Ah, I didn't look closely at `TestNewRepoSpecFromUrl` since it's not a big part of this PR. I'll take a closer look at it, but if it already covers `parseHostSpec` then we don't need to add more. Thanks for the clarification!